RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - removeTeamMember to TS

## Proposed changes

Migrate REST API `teams.removeMember` to TypeScript
## Issue(s)	


## How to test or reproduce

1. Remove member from team
2. Remove member from a team with rooms (that member should be part of those rooms as well)
## Screenshots

## Types of changes




## Checklist



## Further comments
